AllReadyApp/Web-App/AllReady/Areas/Admin/Features/Invite/CreateInviteCommandHandler.cs

```diff
@@ -0,0 +1,90 @@
+﻿using AllReady.Areas.Admin.ViewModels.Invite;
+using AllReady.Models;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AllReady.Areas.Admin.Features.Invite
+{
+    public class CreateInviteCommandHandler : AsyncRequestHandler<CreateInviteCommand>
+    {
+        private AllReadyContext _context;
+
+        public CreateInviteCommandHandler(AllReadyContext context)
+        {
+            _context = context;
+        }
+
+        public Func<DateTime> DateTimeUtcNow = () => DateTime.UtcNow;
+
+        protected override async Task HandleCore(CreateInviteCommand message)
+        {
+            if (message?.Invite == null) throw new ArgumentException("InviteViewModel must be set");
+            if (message?.UserId == null) throw new ArgumentException("User Id must be set");
+            if (string.IsNullOrWhiteSpace(message?.Invite?.InviteeEmailAddress))
```

Comment:
remove this check. InviteeEmailAddress comes from the Send.cshtml view. This view uses the InviteViewModel, which has a required attribute on InviteeEmailAddress
```
[EmailAddress]
[Required]
[Display(Name = "Email")]
public string InviteeEmailAddress { get; set; }
```